package.json

```diff
@@ -29,7 +31,7 @@
     "lint:ts": "yarn tsc --skipLibCheck --noEmit",
     "postinstall": "./scripts/postinstall.sh",
     "start": "react-native start",
-    "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* && rm -rf $TMPDIR/haste-map-* && react-native start -- --reset-cache",
+    "start:clean": "yarn clean && yarn start --reset-cache",
```

Comment:
We def. don't want to remove `node_modules` every time we run `yarn start:clean`

Reply:
This was also reverted.